source/vocab/platform.ttl

```diff
@@ -28,6 +28,7 @@
 
 @prefix : <https://id.kb.se/vocab/> .
 @prefix marc: <https://id.kb.se/marc/> .
+@prefix bulk: <https://id.kb.se/bulk/> .
```

Comment:
Maybe:
```suggestion
@prefix bulk: <https://id.kb.se/ns/bulk/> .
```
to make a (name)space for "other vocabs".

(Could also be under `/vocab/bulk/` ,but that'd make it appear subordinated to KBV rather than orthogonal. The reason for `/ns/` is also to establish a pattern for ontology terms as separate from enumeration lists. These often go under `/term/` _if_ they belong to a concept scheme, but some are directly in the "root".)